src/QirRuntime/samples/StandaloneInputReference/qsharp/qir-standalone-input-reference.qs

```diff
@@ -4,6 +4,28 @@
     open Microsoft.Quantum.Canon;
     open Microsoft.Quantum.Intrinsic;
 
+    function ArrayToString<'T> (array : 'T[]) : String
+    {
+        mutable first = true;
+        mutable itemsString = "[";
+        for item in array
+        {
+            if (first)
+            {
+                set itemsString = itemsString + $"{item}";
+            }
+            else
+            {
+                set itemsString = itemsString + $", {item}";
+            }
+
+            set first = false;
+        }
+
+        set itemsString = itemsString + $"]";
+        return itemsString;
```

Comment:
Why do you mark (with `$` char) the string "]" as an interpolated string? What shall we lose if we remove `$`?

Reply:
There's nothing to lose, I was originally using another approach and did not remove the `$` symbol. Removed it in the latest version.